examples/risk_and_security/retail_agent/README.md

```
@@ -0,0 +1,789 @@
+> **Note**: The PII Defense uses **[Microsoft Presidio](https://github.com/microsoft/presidio)** for detecting and sanitizing personally identifiable information.
+
+**Running with Defenses:**
+
+Run the agent with the defense configuration:
+
+```bash
+nat run --config_file configs/config-with-defenses.yml \
+  --input "Email From: john@email.com\nContent: What garden trowels do you have?"
+```
+
+**Expected Workflow Output (with Defenses)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)